libs/database/src/lib/services/ranker.service.ts

```diff
@@ -118,114 +117,80 @@ export class RankerService {
     excludeContents?: any[]
   ) => {
     console.log('exclude', excludeContents);
```

Comment:
```suggestion
    console.log('exclude', excludeContents);
```

ลบ console.xxx ถ้าจำเป็น เปลี่ยนไปใช้ตัว castcle logger